### PR TITLE
Tabulatin on health/health.c

### PR DESCRIPTION
**Pull request by thiagoftsm**
##### Summary
As @mfundul detected doing a review, we had the file health/health.c with two patterns of tabulation, this PR fixes this problem.
##### Component Name
Health

### CODE REVIEW

File: health/health.c [hunk 1]
```diff
@@ -509,32 +515,30 @@ SILENCE_TYPE check_silenced(RRDCALC *rc, char* host, SILENCERS *silencers) {
...
+             host->hostname,
+             rc->name,
+             (rrdcalc_flags_old & RRDCALC_FLAG_DISABLED)?"true":"false",
+             (rc->rrdcalc_flags & RRDCALC_FLAG_DISABLED)?"true":"false",
+             (rrdcalc_flags_old & RRDCALC_FLAG_SILENCED)?"true":"false",
+             (rc->rrdcalc_flags & RRDCALC_FLAG_SILENCED)?"true":"false"
+        );
+    }
+
+    return (rc->rrdcalc_flags & RRDCALC_FLAG_DISABLED)?1:0;
```
Comment (cakrit) [inline health/health.c]: I would leave this line as it was, since we're supposedly only doing tab changes in this PR. It may help align the rest of the lines as well, because the line numbers change due to this. I don't like the diff that github shows (probably just because of this line), it's misleading.

Comment (thiagoftsm) [reply health/health.c]: You are correct, I return it for the previous code changing only the tabulation.

File: health/health.c [hunk 2]
```diff
@@ -155,7 +155,6 @@ void health_reload_host(RRDHOST *host) {
  * Reload the host configuration for all hosts.
  */
 void health_reload(void) {
-
     rrd_rdlock();
```
Comment (cakrit) [inline health/health.c]: Add that empty line back

File: health/health.c [hunk 3]
```diff
@@ -318,6 +317,13 @@ static inline void health_process_notifications(RRDHOST *host, ALARM_ENTRY *ae)
     health_alarm_execute(host, ae);
 }
 
+/**
```
Comment (cakrit) [inline health/health.c]: Move this comment to another PR

Comment (thiagoftsm) [reply health/health.c]: I removed, I will create a specific for comments on alarm code.

File: health/health.c (hunk 2 above)
Comment (thiagoftsm) [reply health/health.c]: I returned it with another commit, but it will be necessary a PR to remove it, because this function has a pattern with this blank like that is not present in other functions of the file.

**Comment by thiagoftsm** [comment]
> There are two more like the other one, that still cause misalignment and difficulty to compare.
> Basically, see what else need to be moved to another PR, so that all lines align perfectly and we just see tab changes.

There were more than 2 places, so I changed the function call closing and blank line missed.

**Comment by cakrit** [comment]
Still see at the end some line number differences.

**Comment by thiagoftsm** [comment]
> Still see at the end some line number differences.

I got the file from master using the link ( https://raw.githubusercontent.com/netdata/netdata/master/health/health.c ) and I removed the tabulation again from scratch, I saw that github continues giving difference, but this time I was not using a file that was changed before, it is the master branch changed.

**Comment by thiagoftsm** [comment]
> Even `diff` gets confused. I used an online tool and this is fine.

I understand you, I was confused with the fact that after I do what was described in the previous message, I continue having problems with lines.
